cea/inputlocator.py

```diff
@@ -152,7 +153,9 @@ def get_optimization_network_layout_pipes_file(self):
         """scenario/outputs/data/optimization/network/layout/DH_PipesData.csv
         Optimized network layout files for pipes of district heating networks
         """
-        return os.path.join(self.get_optimization_network_layout_folder(), "DH_AllEdges.csv")
+        import cea.config
```

Comment:
can we make `config` a parameter instead of creating it locally? Otherwise, this creates a subtle bug for runs that use a modified config file (e.g. command-line parameters or the arcgis interface)